### PR TITLE
fix(docker): Fix non-root access to volume directories

### LINKED ISSUE #11: Error when trying to create volumes as non-root user
```  
  volumes:
    - ${AIRFLOW_PROJ_HOST_DIR}/dags:/opt/alertflow/dags
    - ${AIRFLOW_PROJ_HOST_DIR}/logs:/opt/alertflow/logs
    - ${AIRFLOW_PROJ_HOST_DIR}/plugins:/opt/alertflow/plugins
```

### PR DESCRIPTION
**Pull request by esloch**
- Closes #11.

### CODE REVIEW

**Comment by luabida** [comment]
@esloch tysm for working on this, gonna review in a minute!

**Comment by luabida** [comment]
@esloch Solved the conflicts, can you check if I did cut something I shouldn't please?

File: docker/Dockerfile [hunk 1]
```diff
@@ -89,6 +89,6 @@ RUN mamba env create -n $ENV_NAME -f /tmp/conda/env-base.yaml \
 
 RUN pip install --root-user-action ignore -r /tmp/airflow-extras.in
 
+WORKDIR /opt/alertflow
```
Comment (luabida) [inline docker/Dockerfile:92]: this is required bc the poetry will search for a pyproject.toml when the script is executed

Comment (esloch) [reply docker/Dockerfile:87]: > this is required bc the poetry will search for a pyproject.toml when the script is executed

Yes, it's here:
https://github.com/esloch/AlertFlow/blob/fix-container-permissions/docker/Dockerfile#L87

File: docker/Dockerfile [hunk 2]
```diff
@@ -33,27 +37,29 @@ RUN apt-get -qq update --yes \
...
+# Create airflow user
+RUN addgroup --gid ${HOST_GID} airflow \
+  && useradd --uid ${HOST_UID} --gid ${HOST_GID} -ms /bin/bash airflow \
+  && echo "airflow ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/airflow \
+  && chmod 0440 /etc/sudoers.d/airflow \
+  && export ENV_NAME="$ENV_NAME" \
   && mkdir -p \
     /opt/conda \
     /opt/alertflow/ \
     /opt/alertflow/dags \
```
Comment (esloch) [inline docker/Dockerfile:56]: @luabida. Tem algo estranho com o seu rebase...
O PR que abri é justamente para corrigir o acesso aos diretórios sources usados pelo service **airflow-init** :
https://airflow.apache.org/docs/apache-airflow/stable/docker-compose.yaml

No seu *compose.yaml* agora o */opt/alertflow* é montado como um volume:
``` 
 volumes:
    &airflow-common-volumes
    - ${AIRFLOW_PROJ_DIR:-.}:/opt/alertflow
``` 
 que logo o *entrypoint.sh* se encarrega de criar os subdiretórios dando as permissões de acesso necessárias:
``` 
ource activate alertadengue
# Give permissions to alertflow user to access working directory sources
mkdir -p /opt/alertflow/logs /opt/alertflow/dags /opt/alertflow/plugins
chown -R "${HOST_UID}:${HOST_GID}" /opt/alertflow/{logs,dags,plugins}
```

**Pull request merged**